global.json

```diff
@@ -1,5 +1,5 @@
 {
   "sdk": {
-    "version": "5.0.100-preview"
+    "version": "5.0.100-preview.7.20366.6"
```

Comment:
Just FYI: this will have a merge conflict with the dev branch: https://github.com/NuGet/NuGet.Client/pull/3591/files